### PR TITLE
Tracking Changes formatting and output fix

### LINKED ISSUE #355: Episode 4: "committing multiple files" formatting needs fix
In https://swcarpentry.github.io/git-novice/04-changes/ , if I reveal the "Committing Multiple Files" exercise and then the solution within that, I see:

`First we make our changes to the mars.txt and venus.txt files: ~~~ $ nano mars.txt $ cat mars.txt ~~~`

and similar display of formatting marks when I should be seeing properly formatted text.

I'll submit a pull request to fix this.

### PR DESCRIPTION
**Pull request by brainwane**
This pull request fixes Markdown formatting for the solution display in the "committing multiple files" exercise, and makes the output more realistic by including the "create mode" output the user would see when adding a new file to the repo.

Fixes #355.

### CODE REVIEW

File: _episodes/04-changes.md [hunk 1]
```diff
@@ -637,11 +637,12 @@ repository (`git commit`):
...
 > > [master cc127c2]
-> > Wrote down my plans to start a base on venus
-> > 2 files changed, 2 insertions(+)
+> >  Wrote down my plans to start a base on venus
+> >  2 files changed, 2 insertions(+)
+> >  create mode 100644 venus.txt
 > > ~~~
 > > {: .output}
-> > ~~~
+> >
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: We can remove this line.

File: _episodes/04-changes.md [hunk 2]
```diff
@@ -637,11 +637,12 @@ repository (`git commit`):
 > > {: .bash}
 > > ~~~
 > > [master cc127c2]
-> > Wrote down my plans to start a base on venus
-> > 2 files changed, 2 insertions(+)
+> >  Wrote down my plans to start a base on venus
```
Comment (rgaiacs) [inline _episodes/04-changes.md]: Why the extra space?

Comment (brainwane) [inline _episodes/04-changes.md]: The extra space is to replicate what the output looks like in the terminal -- at least on my machine, when I make a commit, the first line of the output (i.e., the line starting `[master cc127c2]`) is all the way over to the left, and everything after that is indented by a single space.

Example:

~~~
$ git commit -m "Initial commit"
[master (root-commit) fe775a8] Initial commit
 1 file changed, 11 insertions(+)
 create mode 100644 build-classification.py
~~~

File: _episodes/04-changes.md (hunk 1 above)
Comment (brainwane) [inline _episodes/04-changes.md]: Thanks! I've removed it in the amended commit.

**Comment by brainwane** [comment]
Thanks for the review @rgaiacs -- I've revised the PR and it's ready for fresh review.

**Comment by iglpdc** [comment]
Thanks @brainwane!

**Pull request merged**